VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -92,4 +93,16 @@ private void hideOfflineBar(){
             offlineBar.setVisibility(View.GONE);
         }
     }
+
+    /**
+     * Call this function if you do not want to allow
+     * opening/showing the drawer(Navigation Fragment) on swiping left to right
+     */
+    protected void blockDrawerFromOpening(){
+        DrawerLayout mDrawerLayout = (DrawerLayout)
```

Comment:
You don't need to use ``m`` to prefix variables if they're not instance variables.